heft-plugins/heft-typescript-plugin/src/TypeScriptPlugin.ts

```diff
@@ -360,7 +356,7 @@ export default class TypeScriptPlugin implements IHeftTaskPlugin {
       // watchMode: watchMode,
       maxWriteParallelism: typeScriptConfigurationJson?.maxWriteParallelism || 50,
       scopedLogger: taskSession.logger,
-      emitChangedFilesCallback: (program: IExtendedProgram, changedFiles?: Set<IExtendedSourceFile>) => {
+      emitChangedFilesCallback: (program: TTypescript.Program, changedFiles?: Set<TTypescript.SourceFile>) => {
         // Provide the typescript program dependent plugins
         if (this.accessor.onChangedFilesHook!.isUsed()) {
```

Comment:
Why does this need a non-null assertion? The property should not be optional, since it is created when the accessor is.